.github/CONTRIBUTING.md

```diff
@@ -1,3 +0,0 @@
-The contents of */octicons* */svg* are generated by an automated process. Changes to these files may be accepted, but may also be overwritten.
-
-Octicons are a scalable set of icons created by GitHub. At this time, new icons will only be added when they are needed for GitHub products.
```

Comment:
> At this time, new icons will only be added when they are needed for GitHub products.

Is this statement still accurate or not anymore? It was removed here, and I can't find it re-added anywhere else.